app/src/util/types.ts

```diff
@@ -125,3 +125,15 @@ export enum Wallet {
   MetaMask = 'MetaMask',
   WalletConnect = 'WalletConnect',
 }
+
+export interface Collateral {
```

Comment:
This is almost equal to the `Token` interface. Couldn't we reuse that? `name` might be optional so that existing usages of `Token` don't break.

Reply:
Done!